contrib/devtools/packer/packer_build.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env sh
+
+docker run -v `pwd`:`pwd` -w `pwd` \
```

Comment:
Use `$()` instead of the older backtick notation. It might also make sense to define a local variable instead of calling `pwd` three times.

Reply:
Suggestions applied.